tests/Feature/DemoTest.php

```diff
@@ -24,11 +24,11 @@ class DemoTest extends TestCase
 	public function testDemo0(): void
 	{
 		// save initial value
-		$init_config_value = Configs::get_value('gen_demo_js');
+		$init_config_value = Configs::getValue('gen_demo_js');
 
 		// set to 0
 		Configs::set('gen_demo_js', '0');
-		static::assertEquals('0', Configs::get_value('gen_demo_js'));
+		static::assertEquals('0', Configs::getValue('gen_demo_js'));
```

Comment:
Was Lazy :laughing:

Reply:
Fixed it myself. I considered to make `Configs::getValue` protected, because we do not need it anywhere in productive mode, but the test code still uses it a lot. I did not want to change this now, because this would blow up this PR, but this is something we should consider for another PR.